index.html

```diff
@@ -1212,10 +1252,240 @@
+              graph.getModel().setStyle(cell, style);
+            }
+
+            // Button - Bold
+            var btn = document.createElement("button");
+            btn.innerHTML = 'Bold';
+            btn.setAttribute("id", "boldButton");
+            btn.type = "button";
+            btn.name = "submit";
+            btn.onclick = function() {
+              getFontStyle(graph, cell, 'boldButton', 0);
+            }
+            myform.appendChild(btn);
+
+            // Button - Italics
+            var btn = document.createElement("button");
+            btn.innerHTML = 'Italic';
+            btn.setAttribute("id", "italicButton");
+            btn.type = "button";
+            btn.name = "submit";
+            btn.onclick = function() {
+              getFontStyle(graph, cell, 'italicButton', 1);
+            }
+            myform.appendChild(btn);
+
+            // Button - Underline
+            var btn = document.createElement("button");
+            btn.innerHTML = 'Underline';
+            btn.setAttribute("id", "underlineButton");
+            btn.type = "button";
+            btn.name = "submit";
+            btn.onclick = function() {
+              getFontStyle(graph, cell, 'underlineButton', 2);
+            }
+            myform.appendChild(btn);
+
+            // Line break
+            var linebreak = document.createElement('br');
+            myform.appendChild(linebreak);
+
+            // Line break
+            var linebreak = document.createElement('br');
+            myform.appendChild(linebreak);
+
+            // Button - Submit
+            var btn = document.createElement("button");
+            btn.innerHTML = 'Submit';
+            btn.type = "button";
+            btn.name = "submit";
+
+            // Executes when button 'btn' is clicked
+            btn.onclick = function() {
+                var propertiesObject = {
+                    id: cell.id
+                };
+                for (var key in defaultProperties) {
+                    if (defaultProperties.hasOwnProperty(key)) {
+                        propertiesObject[key] = document.getElementById(key.toString()).value;
+                    }
+                }
+                var style = graph.getModel().getStyle(cell);
+                var styleObject = styleToObject(style);
+                styleObject['fontSize'] = propertiesObject['fontSize'];
+                styleObject['fontFamily'] = propertiesObject['fontFamily'];
+                style = objectToStyle(styleObject);
+                graph.getModel().setStyle(cell, style);
+                graph.getModel().setValue(cell, propertiesObject['text']);
+                wind.destroy();
+            };
+            myform.appendChild(btn);
+
+            // Base heights without fields : 135 px
+            height = 135 + 26 * defaultProperties.length + 15;
+            content.appendChild(myform);
+            var wind = showModalWindow(graph, 'Text and Text font', content, 450, height);
+
```

Comment:
Then can you a write a function called setFontStyle() and call it here? I feel there is redundancy.